.github/workflows/pr.yaml

```diff
@@ -106,7 +111,7 @@ jobs:
           - { name: layer-02-maximus, nick: maximus }
         exclude:
           # Non-clang testing on MacOS is too much work for very little gain
-          - { os: macos-14, compiler: { CC: gcc, CXX: g++ } }
+          # - { os: macos-14, compiler: { CC: gcc, CXX: g++ } }
```

Comment:
The intent behind leaving the code in is as documentation and as an example.
If we ever will want to disable some combination, it's not entirely trivial how to do it, and I'd rather have a comment than spend hours reverse engineering how the exclusion works

Reply:
> The intent behind leaving the code in is as documentation and as an example.
> I'd rather have a comment than spend hours reverse engineering how the exclusion works

This example is already in master/v7. It will be there forever, even after you delete it here. Yes, it might take a minute to find, but that is not really reverse engineering. Moreover, commented out code tends to get stale, requires review/maintenance, and is in the way of other refactoring.

And where do we draw the line? You leave GitHub Actions configuration examples. I add used-to-work C++ code examples. Alice starts fighting for leaving buggy C++ code examples, so that he does not repeat the same mistake twice. Bob starts copying those.... Let's simply remove unused stuff instead.

Reply:
> Alex: Why? How are we going to decide what tests to run for PR branch and what tests to run for staged commits? Please document this important answer in PR description, not (just) this change request comment. We need to reach consensus on test placement scheme before we know which of the two workflow files each test belongs to...

Thank you for updating PR description to answer this question. I edited that answer a little. I will come back to that answer later, to check whether we should move more PR commit tests to PR staging phase, for the same "GitHub cost saving" reason.